_episodes/04-faceting-and-filtering.md

```diff
@@ -15,15 +15,15 @@ objectives:
 keypoints:
 - "You can use facets and filters to explore your data"
 - "You can use facets and filters work with a subset of data in OpenRefine"
-- "You can easily correct common data issues from a Facet"
+- "You can correct common data issues from a Facet"
 ---
 
 ## Facets
 Facets are one of the most useful features of OpenRefine and can help in both getting an overview of the data and to improve the consistency of the data.
 
 A 'Facet' groups all the values that appear in a column, and then allows you to filter the data by these values and edit values across many records at the same time.
 
-The simplest type of Facet is called a 'Text facet'. This simply groups all the text values in a column and lists each value with the number of records it appears in. The facet information always appears in the left hand panel in the OpenRefine interface.
+The simplest type of Facet is called a 'Text facet'. This groups all the text values in a column and lists each value with the number of records it appears in. The facet information always appears in the left hand panel in the OpenRefine interface.
```

Comment:
This still has "The simplest type of Facet", which bugs me. Text facets are perhaps the "most widely applicable type of facet", because they may be used with any type, but that wording is probably still vague to someone learning about facets.

Reply:
How about "One of the most commonly used facets"?